### DISCUSSION TITLE
Reemo API

### DISCUSSION THREAD

**Discussion by vishalkadam47** [Lissy93/dashy#1314] [Ideas]
Is it possible to Integrate Reemo Streaming Technology
Or can you suggest me the best way, I was trying to build my own web client for this
But now as I'm using dashy on a cloud platform, I was looking for a way to manage from a single dashboard will it be possible to integrate and a option to use my API key

 
https://reemo.io/docs

**Comment by vishalkadam47** [comment]
I think its a irrelevant request

**Reply by lissy93** [reply]
You could build a widegt for it, using their API. 
But since it's not something open source or self-hosted, I won't be able to help much.
Dashy probably isn't the application you're looking for, maybe try something more data driven like Grafana
